src/functions/_calc-col-width.scss

```diff
@@ -1,3 +1,12 @@
+
+/// Dynamically calculate the column within the grid
+/// @param {number} $col Number of columns
+/// @param {*} $grid [$grid-grid] - Grid variable to use
+/// @see $grid-grid
+/// @example scss
+///     width: calc-col-width(6, $my-grid);
+
```

Comment:
The empty lines after the documentation varies from 0-2 lines. This should be consistent.